Building AMI with packer 1.6.6. + GP3 Volumes fails

**What steps did you take and what happened:**
[A clear and concise description on how to REPRODUCE the bug.]
Currently default ebs volumes is `gp3`. 
packer 1.6.6 looks for `throughput` var to be set even though it's optional. 
https://github.com/hashicorp/packer/issues/10420

```
make build-ami-amazon-2
hack/ensure-ansible.sh
hack/ensure-ansible-windows.sh
IMPORTANT: Winrm connection plugin for Ansible on MacOS causes connection issues.
See https://docs.ansible.com/ansible/latest/user_guide/windows_winrm.html#what-is-winrm for more details.
To fix the issue provide the enviroment variable 'no_proxy=*'
Example call to build Windows images on MacOS: 'no_proxy=* make build-<target>'
hack/ensure-packer.sh
hack/ensure-goss.sh
Right version of binary present
packer build .......
Error: Failed to prepare build: "amazon-2"

1 error(s) occurred:

* Throughput must be between 125 and 1000 for device /dev/xvda



==> Wait completed after 2 microseconds

==> Builds finished but no artifacts were created.
make: *** [build-ami-amazon-2] Error 1
```

**What did you expect to happen:**
not fail

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

I think the fix goes in v1.7.0. Until then either downgrade to `gp2` or specify `throughput` value

**Environment:**

Project ([Image Builder for Cluster API](https://github.com/kubernetes-sigs/image-builder/tree/master/images/capi), [kube-deploy/imagebuilder](https://github.com/kubernetes-sigs/image-builder/tree/master/images/kube-deploy/imagebuilder), [konfigadm](https://github.com/kubernetes-sigs/image-builder/tree/master/images/konfigadm)):

Additional info for Image Builder for Cluster API related issues:

- OS (e.g. from `/etc/os-release`, or `cmd /c ver`):
- Packer Version:
- Packer Provider:
- Ansible Version: 
- Cluster-api version (if using):
- Kubernetes version: (use `kubectl version`):

/kind bug
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels]

--- a/images/capi/packer/ami/packer.json
+++ b/images/capi/packer/ami/packer.json
@@ -46,7 +46,9 @@
     "snapshot_users": "",
     "subnet_id": "",
     "volume_size": "8",
-    "vpc_id": ""
+    "vpc_id": "",
+    "throughput": "125",
+    "iops": "3000"
   },
   "builders": [
     {
@@ -100,6 +102,8 @@
           "device_name": "{{ user `root_device_name` }}",
           "volume_size": "{{ user `volume_size` }}",
           "volume_type": "gp3",
+          "throughput": "{{ user `throughput` }}",
+          "iops": "{{ user `iops`}}",
           "delete_on_termination": true
         }
       ]

